cmd/nerdctl/container_inspect.go

```diff
@@ -38,13 +38,16 @@ func newContainerInspectCommand() *cobra.Command {
 		SilenceErrors:     true,
 	}
 	containerInspectCommand.Flags().String("mode", "dockercompat", `Inspect mode, "dockercompat" for Docker-compatible output, "native" for containerd-native output`)
+	containerInspectCommand.Flags().BoolP("size", "s", false, "Display total file sizes")
+
 	containerInspectCommand.RegisterFlagCompletionFunc("mode", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		return []string{"dockercompat", "native"}, cobra.ShellCompDirectiveNoFileComp
 	})
 	containerInspectCommand.Flags().StringP("format", "f", "", "Format the output using the given Go template, e.g, '{{json .}}'")
 	containerInspectCommand.RegisterFlagCompletionFunc("format", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		return []string{"json"}, cobra.ShellCompDirectiveNoFileComp
 	})
+
```

Comment:
unnecessary

Reply:
I see.
I guess it is a matter of preference (I am fine with it either way - with a slight preference for separating returns)

Do we have a coding style here?
IIRC, golang-ci has a plugin to enforce these (one way or the other) and maybe we could add it?